docs/config.md

```diff
@@ -0,0 +1,51 @@
+# Configuration
+The philosophy we will use when exposing HCO configuration options is that only
+options that all operators consume can be exposed on the default HCO CR.
+Configuration options that are for specific component operators, will be hidden
+but accepted as valid configuration.
+
+Exposed variables are expressed in the HCO [CSV file](https://github.com/kubevirt/hyperconverged-cluster-operator/blob/dfdd4ac492c1d91c130eec03af1e4f6b04d54c7e/deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml#L18-L20)
+```yaml
+          "spec": {
+            "CDIImagePullPolicy":"IfNotPresent",
+            "KubeVirtImagePullPolicy":"IfNotPresent"
+          }
+```
+
+## 1.0 Config Options
+All component operators must express each of the HCO `Config Options` on their CR.
```

Comment:
Well our common templates CR is not pulling any containers from anywhere.. the options there would not make any sense.

Reply:
What do you mean? You don't deploy any pod? Just curious

Reply:
Exactly that. Common templates do not deploy any pod.. just Templates.